src/plugins/toml/CMakeLists.txt

```
@@ -0,0 +1,102 @@
+function (check_dependencies)
+	set (FOUND_DEPENDENCIES FALSE PARENT_SCOPE)
+
+	find_package (FLEX QUIET)
+	find_package (BISON QUIET)
```

I think you can require Bison 3 here. At least Bison 2.3 is unable to parse the file `parser.y`. This is also one reason why the macOS build jobs fail.

```suggestion
	find_package (BISON 3 QUIET)
```

If you want to make Mac users happy adding something like:

https://github.com/ElektraInitiative/libelektra/blob/f3460ca7bf4371fbde2eb358e5217c9c658c8314/src/plugins/yambi/CMakeLists.txt#L27-L30

before the call to `find_package (BISON)` might also be a good idea.